models/topic-model.js

```diff
@@ -1,12 +1,18 @@
-const db = require('../db/connection')
+const db = require("../db/connection");
+const endpoints = require("../endpoints.json");
+const fs = require("fs/promises");
 
-const fetchAllTopics = () =>{
-    return db.query("SELECT * FROM topics;").then(({rows})=>{
-        return rows;
-        
-    })
-}
+const fetchAllTopics = () => {
+  return db.query("SELECT * FROM topics;").then(({ rows }) => {
+    return rows;
+  });
+};
 
+const fetchAllEndpoints = () => {
```

Comment:
I understand why you'd put this in the model, but as we aren't interacting with the database the functionality for fetchAllEndpoints should go in the controller